bin/agd

```diff
@@ -34,7 +42,9 @@ thisdir=$(cd "$(dirname -- "$real0")" >/dev/null && pwd -P)
 # shellcheck disable=SC1091
 source "$thisdir/../repoconfig.sh"
 
+NO_BUILD=${NO_BUILD-false}
 if test "${1-''}" = build; then
+  NO_BUILD=false
```

Comment:
Would it make sense to normalize this to `true` in the "else" case? I see below that the value of the variable is being executed as a command to terminate a pipeline with the desired status.

Reply:
While I don't want to force it to `true`, it does make sense to normalize it to an executable boolean (one of `true` or `false`).  Thanks for the suggestion!